packages/core/angular/src/wallet/wallet.types.ts

```diff
@@ -1,21 +1,20 @@
-import { Adapter, Wallet, WalletName } from '@solana/wallet-adapter-base';
+import { Adapter, Wallet } from '@solana/wallet-adapter-base';
 import { PublicKey } from '@solana/web3.js';
 
 export interface WalletState {
     wallets: Wallet[];
-    name: WalletName | null;
     wallet: Wallet | null;
     adapter: Adapter | null;
     connecting: boolean;
     disconnecting: boolean;
+    unloading: boolean;
```

Comment:
Is it possible for this to be just an internal implementation detail / private field of the store? I don't think we want to expose it on the state.

Reply:
Agree... It's not really exposed tho, the component store has this behavior where you have to use a selector to access something from the state, given that there's no unloading selector, nothing is really being exposed. Nevertheless, I do think it makes no sense to be part of the state.